package.json

```diff
@@ -8,10 +8,12 @@
     "node": ">=6.13.0"
   },
   "scripts": {
-    "start": "webpack -d --watch & nodemon ./server/index.js",
-    "seed": "node ./database_reviews/fake-data.js"
+    "start": "webpack -d --watch & nodemon ./server/server.js",
+    "seed": "node ./database_reviews/fake-data.js",
+    "test": "jest"
   },
   "dependencies": {
```

Comment:
Have you thought about separating using `devDependencies` for things like webpack and babel? Basically, in your `package.json` file, you want to separate your dependencies that are needed for actually running the application (react, express, body-parser, etc.) from those that are needed for building/compiling your code (babel, webpack) and/or setting up your dev environment (jest, supertest). The dependencies that fall into that second category, go in your `devDependencies`.

Here's some resources that talk about why you might do this and how to achieve it as you bring in new dependencies to your project:

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file
https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c